scanner/src/main/kotlin/scanners/ScanCode.kt

```diff
@@ -243,9 +243,11 @@ class ScanCode(
         listOfNotNull(workingDir, if (Os.isWindows) "scancode.bat" else "scancode").joinToString(File.separator)
 
     override fun transformVersion(output: String) =
-        // "scancode --version" returns a string like "ScanCode version 2.0.1.post1.fb67a181", so simply remove
-        // the prefix.
-        output.substringAfter("ScanCode version ")
+        // "scancode --version" returns a string like "ScanCode version 2.0.1.post1.fb67a181" which might be preceded
+        // by a line saying "Configuring ScanCode for first use...".
+        output.lineSequence().mapNotNull { line ->
+            line.removePrefix("ScanCode version ").takeIf { it != line }
+        }.first()
```

Comment:
A solution without an intermediate list (basically `removePrefix` inlined):
```kotlin
val prefix = "Scancode version "
output.lineSequence().first { it.startsWith(prefix) }.substring(prefix.length)
```

Reply:
Just realized `mapNotNull` on a sequence creates a `TransformingSequence`, not an intermediate list, but I still find this easier to read.

Reply:
Done.